server/sql.go

```diff
@@ -138,6 +140,49 @@ func (s *Server) registerSQLRoutes(g *echo.Group) {
 			return json.Marshal(rowSet)
 		}()
 
+		{
+			errMessage := ""
+			if err != nil {
+				errMessage = err.Error()
+			}
+
+			bytes, err := json.Marshal(api.ActivitySQLQueryHistoryPayload{
+				Statement:    exec.Statement,
+				Duration:     time.Now().UnixMilli() - startTms,
```

Comment:
Our convention is to store time related info as seconds. So if we do want to store duration as ms, please add ms suffix. DurationMs